sdk/wallet-adapter/wallet-kit/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mysten/wallet-kit",
-  "version": "0.3.3",
+  "version": "0.3.4",
```

Comment:
Don't modify the version, create a changeset instead by running `pnpm changeset` in the root.

Reply:
fixed